From 5b959fd8c88ad81bf038068fb1897a7ea32b52b1 Mon Sep 17 00:00:00 2001
From: Toan Nguyen <toan.nguyen.yw@rvc.renesas.com>
Date: Thu, 5 Sep 2019 15:55:19 +0700
Subject: [PATCH 324/405] arm64: defconfig: Enable
 CONFIG_BACKLIGHT_CLASS_DEVICE

This patch changes CONFIG_BACKLIGHT_CLASS_DEVICE from kernel module to
built-in to support LVDS panel in RZ/G2 platform

Signed-off-by: Toan Nguyen <toan.nguyen.yw@rvc.renesas.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 4fdd782..058b6f9 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -234,6 +234,7 @@ CONFIG_DRM_RCAR_DU=y
 CONFIG_DRM_RCAR_DW_HDMI=y
 CONFIG_DRM_RCAR_LVDS=y
 CONFIG_DRM_PANEL_LVDS=y
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_DRM_DUMB_VGA_DAC=y
 CONFIG_DRM_THINE_THC63LVD1024=y
 CONFIG_DRM_I2C_NXP_TDA998X=y
-- 
2.7.4

